ckanext/data_qld_theme/templates/package/resource_read.html

```diff
@@ -37,7 +37,7 @@
 			<ul>
 				{% block resource_actions_inner %}
 				{% if h.check_access('package_update', {'id':pkg.id }) %}
-				<li>{% link_for _('Manage'), named_route='package.resource_edit', id=pkg.name, resource_id=res.id, class_='btn btn-default', icon='wrench' %}</li>
+				<li>{% link_for _('Manage'), controller='package', action='resource_edit', id=pkg.name, resource_id=res.id, class_='btn btn-default', icon='wrench' %}</li>
```

Comment:
can you not have named_route and controller/action?
or is this related to needing to have the plugin released with the named route's and then update the theme?

Reply:
Sometimes you can, but these named routes aren't working. I'm not certain why, and we can look into it when we're properly preparing for CKAN 2.9, but for now we need to fix this up so other changes can go ahead.